src/sentry/grouping/strategies/legacy.py

```diff
@@ -154,38 +161,40 @@ def remove_function_outliers_legacy(function):
 
 
 @strategy(ids=["single-exception:legacy"], interface=SingleException)
-def single_exception_legacy(exception, event, context, **meta):
+def single_exception_legacy(
+    interface: SingleException, event: Event, context: GroupingContext, **meta: Any
```

Comment:
Had to rename `exception/...` to `interface` everywhere because I could not figure out how to type a unnamed positional argument. Defining `__interface` instead of `interface` in the `StrategyFunc` protocol was supposed to work IMO but did not work.

Reply:
Would it make the code (and the diff) more readable to just assign `exception = interface` at the beginning of the function?

Reply:
No strong opinion either, both solutions would probably raise questions with future readers.